auto_nag/scripts/fuzzing_bisection_without_regressed_by.py

```diff
@@ -0,0 +1,108 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.bugzilla import Bugzilla
+
+from auto_nag.bzcleaner import BzCleaner
+
+MAX_DEPTH = 2
+
+
+class FuzzingBisectionWithoutRegressedBy(BzCleaner):
+    def __init__(self):
+        super(FuzzingBisectionWithoutRegressedBy, self).__init__()
+
+    def description(self):
+        return "Bugs with a fuzzing bisection and without regressed_by"
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        data[bugid] = {
+            "is_meta": "meta" in bug["keywords"],
+            "assigned_to": bug["assigned_to"],
+            "assigned_to_nickname": bug["assigned_to_detail"]["nick"],
+            "depends_on": bug["depends_on"],
+        }
+        return bug
+
+    def set_autofix(self, bugs):
+        for bugid, info in bugs.items():
+            self.add_auto_ni(
+                bugid,
+                {"mail": info["assigned_to"], "nickname": info["assigned_to_nickname"]},
+            )
+
+    def _get_bz_params(self, blocked_ids):
+        return {
+            "include_fields": ["assigned_to", "depends_on", "keywords"],
+            "f1": "blocked",
+            "o1": "anyexact",
+            "v1": ",".join(str(bid) for bid in blocked_ids),
+            "f2": "regressed_by",
+            "o2": "isempty",
+            "n3": 1,
+            "f3": "regressed_by",
+            "o3": "everchanged",
+            "n4": 1,
+            "f4": "longdesc",
+            "o4": "casesubstring",
+            "v4": "since this bug contains a bisection range, could you fill (if possible) the regressed_by field",
```

Comment:
I don't know if it's possible, but it'd be nice to get this string from a configuration file from the tool which creates it.